tuftool/src/root.rs

```diff
@@ -286,6 +286,39 @@ impl Command {
         )
         .context(error::SignRoot { path })?;
 
+        // Sanity check of root
```

Comment:
https://gist.github.com/seanmhanson/fe370c2d8bd2b3228680e38899baf5cc

```suggestion
        // Quick check that root is signed by enough key IDs
```